### PR TITLE
Magento_Downloadable: avoid using deprecated escape* methods from Abs…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/composite/fieldset/downloadable.phtml [hunk 1]
```diff
@@ -6,37 +6,42 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 // @deprecated
 ?>
-<?php /* @var $block \Magento\Downloadable\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Downloadable */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/adminhtml/templates/product/composite/fieldset/downloadable.phtml]: Please remove not needed closing and opening PHP tags

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable.phtml [hunk 2]
```diff
@@ -10,6 +10,7 @@
 <?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable.phtml]: Please remove not needed closing and opening PHP tags

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/samples.phtml [hunk 3]
```diff
@@ -9,18 +9,19 @@
 <?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/samples.phtml]: Please remove not needed closing and opening PHP tags

File: app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/creditmemo/name.phtml [hunk 4]
```diff
@@ -6,6 +6,7 @@
 
 /**
  * @var \Magento\Downloadable\Block\Adminhtml\Sales\Items\Column\Downloadable\Name $block
+ * @var \Magento\Framework\Escaper $escaper
  * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
  */
 ?>
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/creditmemo/name.phtml]: Please remove not needed closing and opening PHP tags, and combine all phpdoc blocks to a single one

File: app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/invoice/name.phtml [hunk 5]
```diff
@@ -4,31 +4,34 @@
  * See COPYING.txt for license details.
  */
 
-/** @var \Magento\Downloadable\Block\Adminhtml\Sales\Items\Column\Downloadable\Name $block */
+/**
+ * @var \Magento\Downloadable\Block\Adminhtml\Sales\Items\Column\Downloadable\Name $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/invoice/name.phtml]:
```suggestion
/**
 * @var \Magento\Downloadable\Block\Adminhtml\Sales\Items\Column\Downloadable\Name $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

File: app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/name.phtml [hunk 6]
```diff
@@ -4,33 +4,36 @@
  * See COPYING.txt for license details.
  */
 
-/** @var \Magento\Downloadable\Block\Adminhtml\Sales\Items\Column\Downloadable\Name $block */
+/**
+ * @var \Magento\Downloadable\Block\Adminhtml\Sales\Items\Column\Downloadable\Name $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/name.phtml]: Please remove not needed closing and opening PHP tags and combine all PHP dock blocks

File: app/code/Magento/Downloadable/view/frontend/templates/catalog/product/links.phtml [hunk 7]
```diff
@@ -4,20 +4,25 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /* @var $block \Magento\Downloadable\Block\Catalog\Product\Links */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/frontend/templates/catalog/product/links.phtml]: Please remove not needed closing and opening PHP tags

File: app/code/Magento/Downloadable/view/frontend/templates/email/order/items/creditmemo/downloadable.phtml [hunk 8]
```diff
@@ -4,32 +4,37 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /** @var $block \Magento\Downloadable\Block\Sales\Order\Email\Items\Downloadable */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/frontend/templates/email/order/items/creditmemo/downloadable.phtml]: Please remove not needed closing and opening PHP tags

File: app/code/Magento/Downloadable/view/frontend/templates/email/order/items/invoice/downloadable.phtml [hunk 9]
```diff
@@ -4,33 +4,38 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /** @var $block \Magento\Downloadable\Block\Sales\Order\Email\Items\Downloadable */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/frontend/templates/email/order/items/invoice/downloadable.phtml]: Please remove not needed closing

File: app/code/Magento/Downloadable/view/frontend/templates/email/order/items/order/downloadable.phtml [hunk 10]
```diff
@@ -5,42 +5,46 @@
  */
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 ?>
-<?php /** @var $block \Magento\Downloadable\Block\Sales\Order\Email\Items\Order\Downloadable */ ?>
+<?php
```
Comment (ihor-sviziev) [inline app/code/Magento/Downloadable/view/frontend/templates/email/order/items/order/downloadable.phtml]: Please remove not needed closing and opening PHP tags

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable.phtml]: fixed

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/composite/fieldset/downloadable.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/adminhtml/templates/product/composite/fieldset/downloadable.phtml]: fixed

File: app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/samples.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/adminhtml/templates/product/edit/downloadable/samples.phtml]: fixed

File: app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/creditmemo/name.phtml (hunk 4 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/creditmemo/name.phtml]: fixed

File: app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/invoice/name.phtml (hunk 5 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/invoice/name.phtml]: fixed

File: app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/name.phtml (hunk 6 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/name.phtml]: fixed

File: app/code/Magento/Downloadable/view/frontend/templates/catalog/product/links.phtml (hunk 7 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/frontend/templates/catalog/product/links.phtml]: fixed

File: app/code/Magento/Downloadable/view/frontend/templates/email/order/items/creditmemo/downloadable.phtml (hunk 8 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/frontend/templates/email/order/items/creditmemo/downloadable.phtml]: fixed

File: app/code/Magento/Downloadable/view/frontend/templates/email/order/items/invoice/downloadable.phtml (hunk 9 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/frontend/templates/email/order/items/invoice/downloadable.phtml]: fixed

File: app/code/Magento/Downloadable/view/frontend/templates/email/order/items/order/downloadable.phtml (hunk 10 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Downloadable/view/frontend/templates/email/order/items/order/downloadable.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Functional Tests CE

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests EE, Functional Tests CE

**Pull request closed without merging**